src/trio/_core/_run.py

```diff
@@ -523,15 +523,15 @@ class CancelScope:
     has been entered yet, and changes take immediate effect.
     """
 
-    _cancel_status: CancelStatus | None = attr.ib(default=None, init=False)
-    _has_been_entered: bool = attr.ib(default=False, init=False)
-    _registered_deadline: float = attr.ib(default=inf, init=False)
-    _cancel_called: bool = attr.ib(default=False, init=False)
-    cancelled_caught: bool = attr.ib(default=False, init=False)
+    _cancel_status: CancelStatus | None = attrs.field(default=None, init=False)
+    _has_been_entered: bool = attrs.field(default=False, init=False)
+    _registered_deadline: float = attrs.field(default=inf, init=False)
+    _cancel_called: bool = attrs.field(default=False, init=False)
+    cancelled_caught: bool = attrs.field(default=False, init=False)
 
     # Constructor arguments:
-    _deadline: float = attr.ib(default=inf, kw_only=True)
-    _shield: bool = attr.ib(default=False, kw_only=True)
+    _deadline: float = attrs.field(default=inf, kw_only=True, alias="deadline")
+    _shield: bool = attrs.field(default=False, kw_only=True, alias="shield")
```

Comment:
if it was me I'd perhaps do `from attrs import define, Field, Factory` etc, since we're only using a couple different objects from `attrs`, reuse them a bunch of times, and are widely used enough in the codebase I don't think there'd be much confusion about what library they're from. (and as opposed to `attr.ib` where `ib` on its own doesn't make sense). E.g. in this block it would cut down on the sheer number of characters a decent amount.

But seems plenty fine to stick with `import attrs`